getFeatureFlags/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"os"
+	"fmt"
+	"github.com/joho/godotenv"
+	"log"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/aws"
+
+)
+
+
+type AWSCredentials struct{
+	AccessKey string
+	SecretKey string
+	Region string
+}
+
+var db *dynamodb.DynamoDB
+
+func getAWSCredentials() *AWSCredentials{
+	err := godotenv.Load("../.env")
+
+	if err != nil {
+		log.Fatal("Error loading .env file")
+	}
+	awsCredentials := new(AWSCredentials)
+	if os.Getenv(("ENVIRONMENT")) == "DEVELOPMENT" {
+		awsCredentials.Region = os.Getenv("AWS_REGION")
+		awsCredentials.AccessKey = os.Getenv(("AWS_ACCESS_KEY"))
+		awsCredentials.SecretKey = os.Getenv(("AWS_SECRET_KEY"))
+	}
```

Comment:
wont the code be same even for production? why are we checking for development only here?

Reply:
In case of local we store the env in .env file and passing the file path to `godotenv` package load function, can you explain if it happens the same prod with github secrets